src/aws_secretsmanager_caching/cache/items.py

```diff
@@ -200,7 +200,7 @@ def _execute_refresh(self):
         """
         result = self._client.describe_secret(SecretId=self._secret_id)
         ttl = self._config.secret_refresh_interval
```

Comment:
It's still easily possible to trigger a `TypeError: 'float' object cannot be interpreted as an integer`, would a cast to `int` be appropriate here?